README.md

```diff
@@ -27,9 +27,9 @@ The toolkit provides five separate packages.   See the docs for each action.
 
 Actions are units of work which can either run in a container or on the host machine.
 
-[Choosing an action type](docs/action-types.md): Outlines the differences and why you would want to create a host or a container based action.
+[Choosing an action type](docs/action-types.md): Outlines the differences and why you would want to create a JavaScript or a container based action.
```

Comment:
NOTE to self:  need to add another block encouraging 1 action to 1 repo for market place and versioning to be viable in this model